docs/quickstart.rst

```diff
@@ -393,9 +459,8 @@ configuration module.
 The composition
```

Comment:
Line 448 should refer to line 15 in the code snippet instead of to line 9

Reply:
resolved